packages/fx-core/templates/plugins/resource/aad/auth/tab/README.txt

```diff
@@ -25,7 +25,7 @@ You need to update your application code to take advantage of the SSO authentica
 Debug your application
 -------------------------
 
-1. Right-click your project and select Microsoft 365 Agents Toolkit > Prepare Teams app dependencies
+1. Right-click your project and select Microsoft 365 Agents Toolkit > Prepare app dependencies
```

Comment:
@HuihuiWu-Microsoft , same here.

Reply:
@jayzhang  Is this file still needed? If not, maybe we can remove it.

Reply:
No Prepare Teams app command now. Change to Select Microsoft 365 Account
" Right-click your project and select Microsoft 365 Agents Toolkit >  Select Microsoft 365 Account"